packages/postgress/src/interfaces.ts

```diff
@@ -0,0 +1,4 @@
+export interface ConnectionInfo {
+  fcmNotificationToken?: string
+  maxReceiveBytes?: number
```

Comment:
```suggestion
  sendPushNotification?: (messageId: string) => Promise<void>
  maxReceiveBytes?: number
```

We can use a callback for push notifications that will be implemented on the caller.